packages/back-end/src/services/experimentNotifications.ts

```diff
@@ -170,16 +192,206 @@ export const notifySrm = async ({
     dispatch: async () => {
       if (!triggered) return;
 
-      await dispatchEvent(context, experiment, {
-        type: "srm",
-        experimentId: experiment.id,
-        experimentName: experiment.name,
-        threshold: srmThreshold,
+      await dispatchEvent({
+        context,
+        experiment,
+        event: "warning",
+        data: {
+          object: {
+            type: "srm",
+            experimentId: experiment.id,
+            experimentName: experiment.name,
+            threshold: srmThreshold,
+          },
+        },
       });
     },
   });
 };
 
+type ExperimentSignificanceChange = {
+  experimentId: string;
+  experimentName: string;
+  variationId: string;
+  variationName: string;
+  metricId: string;
+  metricName: string;
+  statsEngine: StatsEngine;
+  criticalValue: number;
+  winning: boolean;
+};
+
+const sendSignificanceEmail = async (
+  experiment: ExperimentInterface,
+  experimentChanges: ExperimentSignificanceChange[]
+) => {
+  const messages = experimentChanges.map(
+    ({ metricName, variationName, winning, statsEngine, criticalValue }) => {
+      if (statsEngine === "frequentist") {
+        return `The metric ${metricName} for variation ${variationName} is
+         ${winning ? "beating" : "losing to"} the baseline and has
+         reached statistical significance (p-value = ${criticalValue.toFixed(
+           3
+         )}).`;
+      }
+      return `The metric ${metricName} for variation ${variationName} has ${
+        winning ? "reached a" : "dropped to a"
+      } ${(criticalValue * 100).toFixed(1)} chance to beat the baseline.`;
+    }
+  );
+
+  try {
+    // send an email to any subscribers on this test:
+    const watchers = await getExperimentWatchers(
+      experiment.id,
+      experiment.organization
+    );
+
+    await sendExperimentChangesEmail(
+      watchers,
+      experiment.id,
+      experiment.name,
+      messages
+    );
+  } catch (e) {
+    logger.error(e, "Failed to send significance email");
+  }
+};
+
+export const notifySignificance = async ({
+  context,
+  experiment,
+  snapshot: currentSnapshot,
+}: {
+  context: Context;
+  experiment: ExperimentInterface;
+  snapshot: ExperimentSnapshotDocument;
+}) => {
+  const lastSnapshot = await getLatestSnapshot(
+    experiment.id,
+    experiment.phases.length - 1
+  );
+
+  if (!lastSnapshot) return;
```

Comment:
Cases:
1. Something broken
2. First snapshot for the experiment.

Test case: make sure we fire on the first snapshot if it is significant on the first run.